h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeNode.java

```diff
@@ -27,9 +27,7 @@
   float predValue = Float.NaN;
   float squaredError = Float.NaN;
   SharedTreeNode leftChild;
-
-
-  public  SharedTreeNode rightChild;
```

Comment:
this is a breaking change

Reply:
Accidental. Reverted, but fixed the formatting.